tests/test_ag3.py

```diff
@@ -1026,44 +1036,57 @@ def test_gene_cnv_frequencies(contig):
         "bf_2012_col_amp",
         "bf_2012_col_del",
```

Comment:
These values depend on which `cohorts` are being provided. Would be better to define e.g., `common_fields` here, then add on what you expect based on what's in the `cohorts`.